core/src/xmake/machine.c

```diff
@@ -131,6 +132,7 @@ static luaL_Reg const g_os_functions[] =
 ,   { "emptydir",       xm_os_emptydir  }
 ,   { "strerror",       xm_os_strerror  }
 ,   { "getwinsize",     xm_os_getwinsize}
+,   { "build_configuration", xm_os_build_configuration}
```

Comment:
It's because I don't wanna one who isn't a developer to use this :smiley: Anyway I'll modify it